### PR TITLE
realm web landing page sync copy fixes

### PR DESCRIPTION
**Pull request by mongodben**
## Pull Request Info

landing page incorrectly said web sdk doesn't support sync

### Jira

- n/a

### Staged Changes

- [Web SDK](https://docs-mongodbcom-staging.corp.mongodb.com/realm/docsworker-xlarge/web_sync_homepage/web)

### Reminder Checklist

If your PR modifies the docs, you might need to also update some corresponding
pages. Check if completed or N/A.

- [ ] Create Jira ticket for corresponding docs-app-services update(s), if any
- [ ] Checked/updated Admin API
- [ ] Checked/updated CLI reference

### Review Guidelines

[REVIEWING.md](https://github.com/mongodb/docs-realm/blob/master/REVIEWING.md)

### CODE REVIEW

**Review by nlarew — APPROVED** [review]

File: source/web.txt [hunk 1]
```diff
@@ -30,10 +30,14 @@ data stored in Atlas and interact with App Services services like
...
-or query data stored in Atlas directly from the browser.
+Web apps built with Realm can query Atlas using the following methods:
+
+- :ref:`Atlas GraphQL API <graphql-api>`
+- The standard MongoDB query API with the :ref:`MongoDB client <web-query-mongodb>`
+- Realm Database API with :ref:`Atlas Device Sync <web-sync>`
+
 Unlike the other Realm SDKs, the Web SDK does not support creating
-a local database or using :ref:`Atlas Device Sync <sync>`.
+a local Realm database.
```
Comment (dacharyc) [inline source/web.txt:40]: I don't think this is actually true anymore. The "Get Started with Device Sync" page says you _can_ create a "local" Realm: https://www.mongodb.com/docs/realm/web/sync/#open-a-non-synced-realm

But it's an in-memory Realm, it doesn't persist to disk. We should probably tweak this wording.

**Review by dacharyc — CHANGES_REQUESTED** [review]
One minor point that doesn't seem quite accurate.

File: source/web.txt (hunk 1 above)
Comment (mongodben) [reply source/web.txt:40]: good call out. adjusting phrasing to: 

```
Unlike the other Realm SDKs, Web SDK only support creating
a in-memory Realm database, not one persisted to disk.
```

File: source/web.txt [hunk 2]
```diff
@@ -30,10 +30,14 @@ data stored in Atlas and interact with App Services services like
...
-Unlike the other Realm SDKs, the Web SDK does not support creating
-a local database or using :ref:`Atlas Device Sync <sync>`.
+Web apps built with Realm can query Atlas using the following methods:
+
+- :ref:`Atlas GraphQL API <graphql-api>`
+- The standard MongoDB query API with the :ref:`MongoDB client <web-query-mongodb>`
+- Realm Database API with :ref:`Atlas Device Sync <web-sync>`
+
+Unlike the other Realm SDKs, the Web SDK only support creating
+a in-memory Realm database, not one persisted to disk.
```
Comment (dacharyc) [inline source/web.txt:40]:
```suggestion
an in-memory Realm database, not one persisted to disk.
```

**Review by dacharyc — APPROVED** [review]
Teeny nit, but otherwise LGTM. Thanks for tweaking this!

**Pull request merged**